src/20241201_Multi_IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances/IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances.md

```diff
@@ -0,0 +1,86 @@
+
+## Specification
+
+| **Market**          | **Asset** | **Current Slope1** | **Recommended Slope1** | **Current Slope2** | **Recommended Slope2** |
+| ------------------- | --------- | ------------------ | ---------------------- | ------------------ | ---------------------- |
+| Ethereum V2         | USDC      | 5.5%               | 12.5%                  | 60%                | -                      |
+| Ethereum V2         | USDT      | 5.5%               | 12.5%                  | 100%               | -                      |
+| Ethereum V2         | DAI       | 5.5%               | 12.5%                  | 75%                | -                      |
+| Ethereum V3 Core    | USDC      | 5.5%               | 12.5%                  | 75%                | 35%                    |
+| Ethereum V3 Core    | USDT      | 5.5%               | 12.5%                  | 75%                | 35%                    |
+| Ethereum V3 Core    | USDe      | 9.0%               | 12.5%                  | 75%                | 50%                    |
+| Ethereum V3 Core    | FRAX      | 5.5%               | 12.5%                  | 75%                | 40%                    |
+| Ethereum V3 Core    | DAI       | 5.5%               | 12.5%                  | 75%                | 35%                    |
+| Ethereum V3 Core    | LUSD      | 5.5%               | 12.5%                  | 87%                | 50%                    |
+| Ethereum V3 Core    | PYUSD     | 5.5%               | 12.5%                  | 80%                | 50%                    |
+| Ethereum V3 Core    | crvUSD    | 5.5%               | 12.5%                  | 80%                | 50%                    |
+| Ethereum V3 Prime   | USDC      | 5.5%               | 12.5%                  | 60%                | 35%                    |
+| Ethereum V3 EtherFi | USDC      | 6.5%               | 12.5%                  | 60%                | 40%                    |
+| Ethereum V3 EtherFi | PYUSD     | 5.5%               | 12.5%                  | 80%                | 40%                    |
+| Ethereum V3 EtherFi | FRAX      | 5.5%               | 12.5%                  | 80%                | 40%                    |
+| Avalanche V3        | USDC      | 5.5%               | 12.5%                  | 60%                | 40%                    |
+| Avalanche V3        | USDT      | 5.5%               | 12.5%                  | 75%                | 40%                    |
+| Avalanche V3        | DAI       | 5.5%               | 12.5%                  | 75%                | 40%                    |
+| Avalanche V3        | FRAX      | 5.5%               | 12.5%                  | 75%                | 40%                    |
+| Polygon V3          | USDC      | 5.5%               | 12.5%                  | 60%                | 40%                    |
+| Polygon V3          | USDC.e    | 10.0%              | 13.5%                  | 80%                | 40%                    |
```

Comment:
Initially, the difference was almost 2 times, and in absolute numbers 4.5% (10 - 5.5).

After this PR, the difference is expected to be reduced to 1%.

Doesn't it seem that It feels more reasonable to keep a higher delta of 2-4?
The same with slope2, it was 60 and 80, and now it's the same 40%

The same about Base USDC, GHO and Gnosis USDC.

Reply:
One of the initiatives of this is to align slope 1 and slope 2 of stablecoins across different markets to avoid brutal volatility of borrow rate observed in recent market conditions. 

The 1% difference of bridged stable coins  is suggested by Chaos, given the slope 1 is already set very high.